dashboard/app/jobs/concerns/active_job_metrics.rb

```diff
@@ -31,18 +31,27 @@ module ActiveJobMetrics
   end
 
   protected def report_job_count
-    Cdo::Metrics.push(
-      METRICS_NAMESPACE, [
+    Cdo::Metrics.push(METRICS_NAMESPACE,
+      # Same metrics as "bin/cron/report_activejob_metrics"
+      [
         {
-          # Same metric as "bin/cron/report_activejob_metrics"
           metric_name: 'JobCount',
-          value: Delayed::Job.count,
+          value: Delayed::Job.where(failed_at: nil).count,
```

Comment:
what do you think about renaming this metric to PendingJobCount, to hopefully clarify that it does not include failed jobs? here and above